docs/configuration.md

```diff
@@ -0,0 +1,15 @@
+# Migration for background images
+
+How the background image is drawn has been changed since version 0.17.
+Intuitively, the background image is now drawn above the background color instead of below it.
+Technically, the background image is no longer blended with the background color.
+
+Before migrating the settings, the background image should be converted to a translucent one with the transparency level matching the original terminal transparency.
+For example, if the original terminal transparency of qtermwidget was 25%, the converted image should have transparency 25%.
```

Comment:
I think the above 2 lines may be a little confusing. For example, the terminal transparency can be zero but the user may prefer a very translucent background image. There's no predetermined relationship between image translucency and terminal transparency. It depends on the image and, of course, the user's taste.

Something like this may be better (only as a suggestion):

```
Any background image can be used but, of course, it should be chosen so that the terminal text can be easily read on it.

Since an image may not be totally dark or light, you might want to use a translucent image as the background. Opaque images can also be converted to translucent ones easily.
```

Reply:
I'd like to provide a clear migration path for the most common existing usage (I assume it is, as many other terminals work that way). How about appending my paragraph after yours as an example?

Reply:
> How about appending my paragraph after yours as an example?

That would be better.

Reply:
Updated, thanks a lot for the suggestion!